test/jasmine/tests/template_test.js

```diff
@@ -141,6 +141,33 @@ describe('makeTemplate', function() {
 
         expect(template).toEqual(expected);
     });
+
+    it('works on DOM element', function(done) {
+        var mock = Lib.extendDeep({}, scatterFillMock);
+        var gd = createGraphDiv();
+
+        Plotly.newPlot(gd, mock)
+      .then(function() {
```

Comment:
weird indentation from here down?

Reply:
I am puzzled. Do you know why it passed the syntax test?

Reply:
I don't think the syntax test cares about method call indentation. It's possible we explicitly disabled that check, perhaps since d3 in particular encourages unusual conventions like
```js
d3.select(cls)
    .attr({...})
    .style({...})
  .select('path')
    .attr({...});
```
but also sometimes it looks best to have all the method calls indented from the initial line and sometime you want them aligned with the initial line.